README.md

```diff
@@ -8,25 +8,81 @@ This repository contains the core of the application and the web services.
+There are some properties that you have to provide. If you can access https://github.com/EBIvariation/configuration/, use those configuration files. If not, save the next snippet in your ~/.m2/settings.xml to use a Maven profile and fill all the properties (fill empty values when needed and change everything starting with "your_"):
+
+```xml
+<settings>
+    <profiles>
+        <profile>
+            <id>production</id>
+            <properties>
+                <dgvapro.host>jdbc:oracle:thin:@your_host:your_port:your_dgva_db</dgvapro.host>
+                <dgvapro.user></dgvapro.user>
+                <dgvapro.passwd></dgvapro.passwd>
+                <eva.evapro.datasource></eva.evapro.datasource>
+                <eva.evapro.jdbc.url>jdbc:postgresql://your_host:your_port/your_eva_db</eva.evapro.jdbc.url>
+                <eva.evapro.user></eva.evapro.user>
+                <eva.evapro.password></eva.evapro.password>
+                <eva.mongo.host></eva.mongo.host>
+                <eva.mongo.user></eva.mongo.user>
+                <eva.mongo.passwd></eva.mongo.passwd>
+                <eva.mongo.auth.db></eva.mongo.auth.db>
+                <eva.mongo.read-preference></eva.mongo.read-preference>
+                <eva.mongo.collections.annotation-metadata></eva.mongo.collections.annotation-metadata>
+                <eva.mongo.collections.annotations></eva.mongo.collections.annotations>
+                <eva.mongo.collections.features></eva.mongo.collections.features>
+                <eva.mongo.collections.files></eva.mongo.collections.files>
+                <eva.mongo.collections.variants></eva.mongo.collections.variants>
+            </properties>
+        </profile>
+    </profiles>
+
+    <servers>
+        <server>
+            <id>maven.oracle.com</id>
+            <username>your_oracle_user</username>
+            <password>your_oracle_encrypted_password</password>
+            <configuration>
+              <basicAuthScope>
+                <host>ANY</host>
+                <port>ANY</port>
+                <realm>OAM 11g</realm>
+              </basicAuthScope>
+              <httpConfiguration>
+                <all>
+                  <params>
+                    <property>
+                      <name>http.protocol.allow-circular-redirects</name>
+                      <value>%b,true</value>
+                    </property>
+                  </params>
+                </all>
+              </httpConfiguration>
+            </configuration>
+      </server>
+   </servers>
+</settings>
+```
+
+After setting up your maven security and maven settings, you can install the whole project and run the tests using the above "production" profile with `mvn clean install -Pproduction`. This will generate the ".war" files that you can deploy in Tomcat or other Java container. You have to set up the JNDI credentials in Tomcat (conf/context.xml), matching the "eva.evapro.datasource" and "eva.evapro.jdbc.url" properties.
```

Comment:
```suggestion
After setting up your Maven security and Maven settings, you can install the whole project and run the tests using the above "production" profile with `mvn clean install -Pproduction`. This will generate the ".war" files that you can deploy in Tomcat or other Java container. You have to set up the JNDI credentials in Tomcat (conf/context.xml), matching the "eva.evapro.datasource" and "eva.evapro.jdbc.url" properties.
```